### PR TITLE
Rn entry 2023.01.2

### PR DESCRIPTION
**Pull request by isaacdurham-okta**
## Description:
- **What's changed?** Release Note Entries
- **Is this PR related to a Monolith release?** 2023.01.2

### Resolves:

* [OKTA-568344](https://oktainc.atlassian.net/browse/OKTA-568344)

### CODE REVIEW

File: packages/@okta/vuepress-site/.vuepress/config.js [hunk 1]
```diff
@@ -215,7 +215,7 @@ module.exports = ctx => ({
         items: [
           { text: 'Pricing', link: 'https://www.okta.com/pricing/#customer-identity-products' },
           { text: 'Integrate with Okta', link: '/okta-integration-network/' },
-          { text: 'Change log', link: '/docs/release-notes/' },
+          { text: 'Release Notes', link: '/docs/release-notes/' },
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/.vuepress/config.js:218]: I'm sure this is fine, but can you run this update by Chelsea, Kim, and Janet? In the past, I remember they really wanted to cling onto the API "change log" terminology.

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md [hunk 2]
```diff
@@ -6,11 +6,44 @@ title: Okta Identity Engine API Products release notes 2023
...
+| [Interaction Code flow supports native SSO](#interaction-code-flow-supports-native-sso)                          |February 1, 2023 |
+| [Bugs fixed in 2023.01.2](#bugs-fixed-in-2023-01-2)                                                              |February 1, 2023 |
+
+#### Clock skew for access and ID tokens
+
+A 30 second clock skew is now allowed for access and ID tokens to validate that a token was created before the current time. <!--OKTA-538956 -->
+
+#### Content Security Policy update
+
+To improve our security and protect against Cross-Site Scripting (XSS) attacks, we are expanding the use of [Content Security Policy (CSP)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) to  include when requests come from custom domains on non-customizable pages. <!--OKTA-567669 -->
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:24]:
```suggestion
To improve our security and protect against Cross-Site Scripting (XSS) attacks, we are expanding the use of [Content Security Policy (CSP)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) to include when requests come from custom domains on non-customizable pages. <!--OKTA-567669 -->
```

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md [hunk 3]
```diff
@@ -6,11 +6,44 @@ title: Okta Identity Engine API Products release notes 2023
...
+
+#### Interaction Code flow supports Native SSO
+
+The Interaction Code flow now supports the device_sso scope that you can use to perform Native SSO. This scope allows you to obtain an interaction code and exchange it for tokens and a device secret. See [Implement authorization by grant type](https://developer.okta.com/docs/guides/implement-grant-type/interactioncode/main/). <!--OKTA-551724 -->
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:28]:
```suggestion
The Interaction Code flow now supports the `device_sso` scope that you can use to perform Native SSO. This scope allows you to obtain an interaction code and exchange it for tokens and a device secret. See [Implement authorization by grant type](https://developer.okta.com/docs/guides/implement-grant-type/interactioncode/main/). <!--OKTA-551724 -->
```

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md [hunk 4]
```diff
@@ -6,11 +6,44 @@ title: Okta Identity Engine API Products release notes 2023
...
+
+#### Bugs fixed in 2023.01.2
+
+* Requests made with an empty Origin header returned a 500 error response. (OKTA-449621)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:32]:
```suggestion
* Requests made with an empty Origin header returned an HTTP 500 error response. (OKTA-449621)
```

File: packages/@okta/vuepress-site/docs/release-notes/2023/index.md [hunk 5]
```diff
@@ -4,6 +4,36 @@ title: Okta API Products release notes 2023
...
+| [Content Security Policy update](#content-security-policy-update)                                                |February 1, 2023 |
+| [Bugs fixed in 2023.01.2](#bugs-fixed-in-2023-01-2)                                                              |February 1, 2023 |
+
+#### Clock skew for access and ID tokens
+
+A 30 second clock skew is now allowed for access and ID tokens to validate that a token was created before the current time. <!--OKTA-538956 -->
+
+#### Content Security Policy update
+
+To improve our security and protect against Cross-Site Scripting (XSS) attacks, we are expanding the use of [Content Security Policy (CSP)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) to  include when requests come from custom domains on non-customizable pages. <!--OKTA-567669 -->
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023/index.md:21]:
```suggestion
To improve our security and protect against Cross-Site Scripting (XSS) attacks, we are expanding the use of [Content Security Policy (CSP)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) to include when requests come from custom domains on non-customizable pages. <!--OKTA-567669 -->
```

File: packages/@okta/vuepress-site/docs/release-notes/2023/index.md [hunk 6]
```diff
@@ -4,6 +4,36 @@ title: Okta API Products release notes 2023
...
+
+#### Bugs fixed in 2023.01.2
+
+* Requests made with an empty Origin header returned a 500 error response. (OKTA-449621)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023/index.md:25]:
```suggestion
* Requests made with an empty Origin header returned an HTTP 500 error response. (OKTA-449621)
```

**Review by vanngo-okta — APPROVED** [review]
Looks good! Some nits, approving to not hold you up. 👍

File: packages/@okta/vuepress-site/.vuepress/config.js (hunk 1 above)
Comment (isaacdurham-okta) [reply packages/@okta/vuepress-site/.vuepress/config.js:218]: Slack signoff by Janet.

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md (hunk 2 above)
Comment (isaacdurham-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:24]: Adjusted

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md (hunk 3 above)
Comment (isaacdurham-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:28]: Adjusted

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md (hunk 4 above)
Comment (isaacdurham-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:32]: Adjusted

File: packages/@okta/vuepress-site/docs/release-notes/2023/index.md (hunk 6 above)
Comment (isaacdurham-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023/index.md:25]: Adjusted

File: packages/@okta/vuepress-site/docs/release-notes/2023/index.md (hunk 5 above)
Comment (isaacdurham-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023/index.md:21]: Adjusted

**Pull request merged**